cves/kernel/CVE-2021-3659.yml

```diff
@@ -356,8 +361,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: the bug is a null check
```

Comment:
Possibly elaborate a little more that the vulnerability was caused by a missing null check that can lead to a null pointer dereference.